plugins/optimization-detective/class-od-link-collection.php

```diff
@@ -215,9 +260,16 @@ public function get_response_header(): ?string {
 	/**
 	 * Counts the links.
 	 *
-	 * @return int Link count.
+	 * @return int<0, max> Link count.
 	 */
 	public function count(): int {
-		return count( $this->links );
+		return array_sum(
```

Comment:
Use case for `array_reduce` perhaps?

Reply:
I tried this:

```php
		return array_reduce(
			array_values( $this->links_by_rel ),
			static function ( int $accumulator, array $links ): int {
				return $accumulator + count( $links );
			},
			0
		);
```

But then PHPStan complains that it returns an `int` instead of a `positive-int`.